### PR TITLE
Re-organize and expand LEDs page

### PR DESCRIPTION
**Pull request by Hwurzburg**
addressing #2423

### CODE REVIEW

File: common/source/docs/common-external-leds.rst [hunk 1]
```diff
@@ -4,20 +4,44 @@
...
+Serially Connected Devices
+==========================
+
+Currently, Ardupilot supports the Neopixel style RGB LED and strings. They can be used for NTF notifications from the autopilot on status and warnings like other RGB LEDs, or be programmed in unlimited ways using LUA scripts on the autopilot's SD card. See :ref:`common-lua-scripts` for more use examples using LUA scripts.
+
+
+.. image:: ../../../images/neopixel-string.jpg
+
+A "NeoPixel" style (WS2812B compatible) RGB LED can be attached to any GPIO capable output by setting its SERVOx_FUNCTION to one of the ``NeoPixelx`` output functions and setting :ref:`NTF_LED_TYPES<NTF_LED_TYPES>` parameter to ``NeoPixel``. Multiple ``NeoPixelx`` output functions are provided for connecting multiple strings (up to 4).
```
Comment (eovnu87435ds) [inline common/source/docs/common-external-leds.rst]: Is it worth clarifying that with the neopixels, you need to keep the necessary pin assigned as a PWM output? 
see: [this thread.](https://discuss.ardupilot.org/t/neopixel-ws2812b-not-working-on-omnibus-f4-pro/51293)

Comment (Hwurzburg) [reply common/source/docs/common-external-leds.rst]: see the second paragraph (highlited note) on the page which addresses this topic

**Comment by rmackay9** [comment]
LGTM

**Comment by Hwurzburg** [comment]
added notes on Neopixel powering

**Pull request merged**